From 0c4efcf268d5f7f940007390efa96ccec8e64c96 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Tue, 20 Jul 2021 13:43:44 +0200
Subject: [PATCH 1/2] drivers: add PCIe driver for Mediatek MT7915E

This adds Mediatek MT7915E miniPCIe [1] to Updater Drivers.
If there is a plugged mini PCIe card with the same vendor
and device ID in any Turris device, then it will install
kmod-mt7915e once there is a working Internet connection.

[1] https://www.asiarf.com/shop/hot-sales/wifi6-4t4r-dual-bands-selectable-mpcie-card-ieee802-11ax-ac-a-b-g-n-2-4g-5ghz-aw7915-np1/
---
 drivers/pci.lua | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/pci.lua b/drivers/pci.lua
index 35c7c03..2176d25 100644
--- a/drivers/pci.lua
+++ b/drivers/pci.lua
@@ -26,6 +26,11 @@ local db = {
 		device = 0x7612,
 		packages = {"kmod-mt76"}
 	},
+	{ -- MEDIATEK Corp. MT7915E 802.11ax PCI Express Wireless Network Adapter
+		vendor = 0x14c3,
+		device = 0x7915,
+		packages = {"kmod-mt7915e"}
+	},
 }
 
 ----------------------------------------------------------------------------------
-- 
2.30.2

